app/src/main/java/org/mozilla/focus/browser/DisplayToolbar.kt

```diff
@@ -22,6 +27,17 @@ class DisplayToolbar(
 
     @Suppress("MagicNumber") // A mathematical expression - No need to add constants for 100% and 50%.
     override fun onOffsetChanged(appBarLayout: AppBarLayout, verticalOffset: Int) {
+
+        val accessibilityManager = context.getSystemService(Context.ACCESSIBILITY_SERVICE) as AccessibilityManager
+        if (accessibilityManager.isTouchExplorationEnabled) {
```

Comment:
We should also check for switch services.
Please refer to  
https://github.com/mozilla-mobile/fenix/blob/2b363b98683a92d8f51af3757b899294908d9308/app/src/main/java/org/mozilla/fenix/utils/Settings.kt#L605-L615 & 
https://github.com/mozilla-mobile/fenix/blob/2b363b98683a92d8f51af3757b899294908d9308/app/src/main/java/org/mozilla/fenix/utils/Settings.kt#L688-L705